db/test_data.sql

```diff
@@ -0,0 +1,239 @@
+INSERT INTO client(id, name)
```

Comment:
Why not use loop ? http://stackoverflow.com/questions/19145761/postgres-for-loop